terracumber_config/tf_files/SUSEManager-4.1-PRV.tf

```diff
@@ -135,7 +135,7 @@ module "cucumber_testsuite" {
       image = "sles15sp1"
       name = "min-sles15"
       provider_settings = {
-        mac = "52:54:00:00:00:33"
```

Comment:
This mac address was wrong, as `52:54:00:00:00:33` was for `suma-ref41-min-sles15` (and now for `suma-ref41-cli-sles15`)